lambdas/data-migration2/tests/test-pdrs.js

```diff
@@ -0,0 +1,328 @@
+const test = require('ava');
+const omit = require('lodash/omit');
```

Comment:
I think you're going to get a lint error for importing this and not using it, but we'll see

Reply:
You are probably right 😬